### PR TITLE
Cypress issue#92: cy.contains() doc.

### PR DESCRIPTION
**Pull request by sainthkh**
## Todo

- [x] Do not merge until cypress-io/cypress#5653 is merged

Related to cypress-io/cypress#92, cypress-io/cypress#5653

### CODE REVIEW

**Comment by sainthkh** [comment]
Updated:
* `matchCase` example
* History
* Note about `pre` exception

File: source/api/commands/contains.md [hunk 1]
```diff
@@ -186,6 +187,19 @@ cy.get('form')                  // yields <form>...</form>
 
 Without the explicit selector the subject would change to be the `<button>`. Using the explicit selector ensures that chained commands will have the `<form>` as the subject.
 
+## Case Sensitivity
+
+Here's an example about `matchCase` option to ignore case sensitivity.
```
Comment (jennifer-shehane) [inline source/api/commands/contains.md]:
```suggestion
Here's an example using the `matchCase` option to ignore case sensitivity.
```

File: source/api/commands/contains.md [hunk 2]
```diff
@@ -228,6 +242,35 @@ cy.contains('Delete User').click()
 cy.contains('Yes, Delete!').click()
 ```
 
+## Leading, trailing, duplicate whitespaces aren't ignored in `<pre>` tag
+
+Unlike other tags, `<pre>` doesn't ignore leading, trailing, duplicate whitespaces like below:
```
Comment (jennifer-shehane) [inline source/api/commands/contains.md]:
```suggestion
Unlike other tags, `<pre>` doesn't ignore leading, trailing, or duplicate whitespaces as shown below:
```

**Comment by jennifer-shehane** [comment]
⚠️  Do not merge until cypress-io/cypress#5653 is merged

**Comment by chrisbreiding** [comment]
@sainthkh The feature issue has been merged, so this is ready to go in as well. Can you fix the merge conflicts? I tried to do it myself, but it wouldn't let me push my changes.

**Comment by jennifer-shehane** [comment]
Yes! awesome.

**Comment by jennifer-shehane** [comment]
Ok, yah I don't have permission to push either. 

@sainthkh Please check the "Allow edits from maintainers" checkbox when submitting your PRs to docs. This will make it easier for the maintainers to make minor adjustments, to help with tests or any other changes we may need.

File: source/guides/references/migration-guide.md [hunk 3]
```diff
@@ -190,3 +191,17 @@ module.exports = (on) => {
...
+Browsers ignore leading, trailing, duplicate whitespaces. And Cypress now does that, too.
+
+```html
+<p>hello
+world</p>
+```
+
+```javascript
+cy.get('p').contains('hello world') // Fail in 3.x. Pass in 4.0.
+cy.get('p').contains('hello\nwolrd') // Pass in 3.x. Fail in 3.x.
```
Comment (jennifer-shehane) [inline source/guides/references/migration-guide.md]: Caught a typo giving this another look over.

```suggestion
cy.get('p').contains('hello\nworld') // Pass in 3.x. Fail in 3.x.
```

**Comment by sainthkh** [comment]
Resolved conflict.

**Pull request merged**